### DISCUSSION TITLE
Check activated Layers in torch backend

### DISCUSSION THREAD

**Discussion by Pacman1984** [ThilinaRajapakse/simpletransformers#939] [Q&A]
Hi, 

is there a way to check the activated layers in the torch backend?

If i build a model from the ground with transformers and torch i usually do somethinglike this:

```
for name, param in model.named_parameters():
    print(f'{name}: {param.requires_grad}')
```

and it will show me the activated (trainable) layers.

Best

**Comment by ThilinaRajapakse** [comment]
You can do the same thing by accessing the `model` attribute of any Simple Transformers model. Alternatively, you can just use the `get_named_parameters()` method of `ClassificationModel`, `NERModel`, etc.
